runtime/src/snapshot_config.rs

```diff
@@ -31,3 +30,22 @@ pub struct SnapshotConfig {
     /// NOTE: Incremental snapshots will only be kept for the latest full snapshot
     pub maximum_incremental_snapshot_archives_to_retain: usize,
 }
+
+impl Default for SnapshotConfig {
+    fn default() -> Self {
+        Self {
+            full_snapshot_archive_interval_slots:
+                snapshot_utils::DEFAULT_FULL_SNAPSHOT_ARCHIVE_INTERVAL_SLOTS,
+            incremental_snapshot_archive_interval_slots:
+                snapshot_utils::DEFAULT_INCREMENTAL_SNAPSHOT_ARCHIVE_INTERVAL_SLOTS,
+            snapshot_archives_dir: PathBuf::default(),
```

Comment:
is this == snapshot_archives_dir: PathBuf::from("/"),?

Reply:
Good catch! No, it's basically an empty string, like `String::new()`, which ends up being `Vec::new()`.

In the rpc_service test that sets snapshot_archive_dir and bank_snapshots_dir to `PathBuf::from("/")`, the values don't matter at all, just that SnapshotConfig is `Some` and not `None`.